src/node_url.cc

```diff
@@ -144,7 +144,6 @@ URLHost::~URLHost() {
 
 #define ERR_ARGS(XX)                                                          \
   XX(ERR_ARG_FLAGS)                                                           \
-  XX(ERR_ARG_INPUT)                                                           \
```

Comment:
I think it would make sense to simplify this a lot further. It’s odd to define function parameters this way, and the fewer parameters the callback takes, the odder it becomes. I think you can drop `ERR_ARGS` and `url_error_cb_args` entirely.